api/tests/test_security_user.py

```diff
@@ -29,35 +29,12 @@ def setUp(self):
 
         self.access_token = json.loads(response.content)["access_token"]
 
-    def test_app_can_get_security_users_emails(self):
+    def test_get_security_users_emails_endpoint_doesnt_exist(self):
```

Comment:
I don't think this test is necessary. Taking off the previous ones will do. Tests show the app's functionality. So when I don't see this endpoint being consumed in the tests, I know it's not available.

Reply:
@kmazi Alright, let me remove it